### DISCUSSION TITLE
Netlify Preview Actions

### DISCUSSION THREAD

**Discussion by at-the-vr** [AccessibleForAll/AccessibleWebDev#389] [Ideas]
I believe its a good time to add Netlify (or Vercel) preview github actions. 
- helps Reviewers catching up to PR
- gives PR creators to identify points of improvements

**Comment by EmmaDawsonDev** [comment]
Thanks for the idea. The project does use the netlify deploy preview but not on the main branch because the project is currently using the free netlify tier and linking it to netlify previews rapidly eats up the limited build minutes since it rebuilds the deploys on every commit once a PR is made. Therefore, I've set the project up so that the deploy previews only trigger when a PR is made to the production branch instead.

**Comment by at-the-vr** [comment]
got it 👍 as long as its covered by maintainers I think its good 😉
